docs-site/src/markdown-pages/work-in-progress/plan-limits.md

```diff
@@ -4,69 +4,62 @@ section: In Progress
 title: Plan Limits Viewer
 ---
 
-This page describes the work currently in progress around for the the Plan Limits viewer. 
+You can check out the Plan Limits Viewer at [https://plan-limits-viewer.gw.govt.nz/](https://plan-limits-viewer.gw.govt.nz/)
 
----
+![A screenshot of the Plan Limits Viewer](./plan-limits-viewer-screenshot.png)
 
 ## Context
 
-Regional Councils create plans for their regions. They set out the objectives, policies and methods for people and organisations that use the Region’s resources. These plans contain measurable limits (or targets) that can be compared to environmental observations or consents data to help understand how the council is performing against these limits.
+Regional Councils create plans which set out objectives and policies that govern usage of that Region’s resources. These plans contain measurable limits or targets that can be compared to environmental observations or consents data to help understand how the council is performing against these limits.
 
-These plans a generally mastered in text based documents, i.e. PDF files with Geospatial boundaries coming from Council GIS systems. This means that for a user, it requires parsing large amounts of text to identify / understand what limits apply to which areas. This has a secondary effect that means it is difficult to build digital solutions that can compare these limits to other data sets.
+These plans are generally published in text based formats such PDF files, referencing Geospatial boundaries coming from Council GIS systems. For end-users, this means large amounts of text often containing complex rules need to be read and understood to know what limits apply to which areas. This format also makes it is difficult to build digital solutions that can compare these limits to other data sets.
 
-The Plan Limits software system pulls together the information from the PDF documents and the GIS systems allowing a user to easily understand what limits apply to a geospatial feature.
+The Plan Limits Viewer digitises these rules alongside the GIS data they reference, allowing a user to easily understand what limits apply where.
```

Comment:
In this section should we maybe include a bit about the future of integrating other councils?